tests/system/base.py

```diff
@@ -41,6 +44,14 @@ def setUp(self) -> None:
         )
         self.client = self.create_client(ADMIN_USERNAME, ADMIN_PASSWORD)
 
+    def load_example_data(self) -> None:
+        self.datastore.truncate_db()
+        example_data = json.loads(requests.get(self.EXAMPLE_DATA).content)
+        for collection, models in example_data.items():
+            for model in models:
+                fields = {k: v for k, v in model.items() if k != "id"}
```

Comment:
Why do you not write the id into the datastore? for some tests it's needed

Reply:
It just because we do not do this in most (or all) other tests. I have no problem if you do so here.

Reply:
It's added automatically in the `create_model` method for the above reason. I think it's better to not filter it out as it might create hard-to-debug errors.

Reply:
Ok.